.github/workflows/upload-tutorials-stats.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Upload tutorials stat
+
+# Controls when the workflow will run
+on:
+  pull_request:
```

Comment:
You probably want to run it on pull request only if it modifies particular files(otherwise this job will run for every PR, which is not something we want, do we?)

Reply:
yes, this is for testing only. Will remove before merging.